### PR TITLE
Portfolio menu

### PR DESCRIPTION
**Pull request by forLoop94**
1. I added a modal for my hamburger.
2. When the user clicks (or taps) the hamburger button, the mobile menu appears.
3. When the user clicks (or taps) the close (X) button, the mobile menu disappears.
4. When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
5. When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -11,13 +11,14 @@
...
-  <main>
+  <main data-main>
     <header class="flex">
       <a href="#" class="logo">My Logo</a>
       <nav>
-        <div id="menu-btn">
+        <div id="menu-btn" data-menu>
           <div class="menu-bar"></div>
           <div class="menu-bar"></div>
           <div class="menu-bar"></div>
```
Comment (ShadyShawkat) [inline index.html:24]: - [OPTIONAL] Kindly make sure that the links in the navbar on the desktop view are working as well and redirect the user to the right section of the page.

File: index.html [hunk 2]
```diff
@@ -11,13 +11,14 @@
   <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
   <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
   <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+  <script defer src="index.js"></script>
 </head>
 <body>
-  <main>
+  <main data-main>
     <header class="flex">
       <a href="#" class="logo">My Logo</a>
```
Comment (ShadyShawkat) [inline index.html:19]: - [ ] Please eliminate the extra white space at the bottom of the mobile menu to match the given design.
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by forLoop94** [comment]
1. The links in the navbar on the desktop view are now working and redirect the user to the right section of the page.
2. I have eliminated the white space at the bottom of the modal.

Thank you.

File: styles.css
Comment (AbelG101) [inline styles.css:1]: https://github.com/forLoop94/Personal-Portfolio_project/blob/3c8d937be156e52e46b9222bfeec1111a65c78b4/styles.css#L35-L46
- [ ] [OPTIONAL] You're doing a really great job here but it seems like your logo is overflowing from the navbar I'd suggest you remove the `width: 70px;` from the above code as it's making your portfolio look unprofessional 👶.
- Here's how it looks now:
firefox_PrhoKt99QD

**Review by AbelG101 — APPROVED** [review]
# Hi @forLoop94 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- UI looking on point ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by forLoop94** [comment]
I will fix the overflowing logo.
Thank you.

**Pull request merged**